.pre-commit-config.yaml

```diff
@@ -0,0 +1,7 @@
+- repo: git://github.com/antonbabenko/pre-commit-terraform
+  rev: v1.19.0
+  hooks:
+    - id: terraform_fmt
```

Comment:
Have you added details on how to install pre-commit and terraform cli to the main readme?

Reply:
added file to docs/runbooks